docs/installation.md

```diff
@@ -1,9 +1,16 @@
 # Installing Ruff
 
-Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI:
+Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI. We recommend installing Ruff with [uv](https://docs.astral.sh/uv/):
 
 ```console
+$ # With uv.
+$ uv tool install ruff
```

Comment:
Is it our recommended setup to install ruff globally vs as a project dependency? Having Ruff pinned seems like a big advantage to me and suggesting to install it globally feels like we're misguiding users to struggle with random CI failures etc. in the future

Reply:
I think we could recommend both which is to either install it globally or add it as a project dependency with a specific version. Rooster should take care of updating the versions in the documentation (https://github.com/astral-sh/ruff/blob/d45c1ee44f53d9c606085e79da1830b558fc001f/pyproject.toml#L106-L114).